automation/test-linux.sh

```diff
@@ -63,43 +77,60 @@ fi
 
 delete_vm(){
   vm_name=$1
-  local template_name=$2
+
+  if [ "${KUBE_CMD}" == "oc" ]; then
+      echo $KUBE_CMD
+      template_option=$2
+  elif [ "${KUBE_CMD}" == "kubectl" ]; then
+      echo $KUBE_CMD
+      template_option="-f $2"
+      template_local="--local"
+  fi
+
   set +e
   #stop vm
   ./virtctl stop $vm_name -n $namespace
   #delete vm
-  oc process -n $namespace -o json $template_name NAME=$vm_name SRC_PVC_NAME=$TARGET-datavolume-original SRC_PVC_NAMESPACE=kubevirt | \
-    oc delete -n $namespace -f -
+  oc process ${template_local} -n $namespace -o json $template_option NAME=$vm_name SRC_PVC_NAME=$TARGET-datavolume-original SRC_PVC_NAMESPACE=kubevirt | \
+    ${KUBE_CMD} delete -n $namespace -f -
   set -e
   #wait until vm is deleted
-  while oc get -n $namespace vmi $vm_name 2> >(grep "not found") ; do sleep $sample; done
+  while ${KUBE_CMD} get -n $namespace vmi $vm_name 2> >(grep "not found") ; do sleep $sample; done
 }
 
 run_vm(){
   vm_name=$1
   template_path="dist/templates/$vm_name.yaml"
-  local template_name=$( oc get -n ${namespace} -f ${template_path} -o=custom-columns=NAME:.metadata.name --no-headers -n kubevirt )
+  local template_name=$( ${KUBE_CMD} get -n ${namespace} -f ${template_path} -o=custom-columns=NAME:.metadata.name --no-headers -n kubevirt )
   running=false
 
   # add cpumanager=true label to all worker nodes
   # to allow execution of tests using high performance profiles
-  oc label nodes -l node-role.kubernetes.io/worker cpumanager=true --overwrite
+  ${KUBE_CMD} label nodes -l node-role.kubernetes.io/worker cpumanager=true --overwrite
 
   #If first try fails, it tries 2 more time to run it, before it fails whole test
   for i in `seq 1 3`; do
     error=false
-    oc process -n $namespace -o json $template_name NAME=$vm_name SRC_PVC_NAME=$TARGET-datavolume-original SRC_PVC_NAMESPACE=kubevirt | \
+    if [ "${KUBE_CMD}" == "oc" ]; then
```

Comment:
Can you please move this if statement outside of loop?

Reply:
Done